pkg/controlplane/instance.go

```diff
@@ -647,7 +650,7 @@ var (
 		nodev1beta1.SchemeGroupVersion, // remove in 1.26
 		policyapiv1beta1.SchemeGroupVersion,
 		storageapiv1beta1.SchemeGroupVersion,
-		flowcontrolv1beta1.SchemeGroupVersion,
+		flowcontrolv1beta1.SchemeGroupVersion, // TODO: remove in 1.26
 		flowcontrolv1beta2.SchemeGroupVersion,
```

Comment:
@deads2k @liggitt  I think I need to add `flowcontrolv1beta3.SchemeGroupVersion` here as well after reading the KEP https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/3136-beta-apis-off-by-default, 

But I had expected a test to fail due to this mis-configuration of the new beta version, I have added extra check to the test for this KEP:  please review https://github.com/kubernetes/kubernetes/pull/112402

Reply:
Yes, it should be added here and the resource enabled by name.